akka-docs/src/main/paradox/stream/operators/Source/queue.md

```diff
@@ -1,15 +1,35 @@
 # Source.queue
 
-Materialize a `SourceQueue` onto which elements can be pushed for emitting from the source. 
+Materialize a `SourceQueue` or `BoundedSourceQueue` onto which elements can be pushed synchronously or asynchronously for emitting from the source.
 
 @ref[Source operators](../index.md#source-operators)
 
-## Signature
+## Signature (synchronous)
```

Comment:
Maybe let's use the name of the materialized queue as the identifier for the two variants?

```suggestion
## Signature (`BoundedSourceQueue`)
```

Reply:
:+1: